tools/scripts/generate.sh

```diff
@@ -23,6 +23,7 @@ echo "# Running transformation based on $OPENAPI_FILE_NAME to the $transformed_f
 cp "$OPENAPI_FOLDER/$OPENAPI_FILE_NAME" "$openapiFileLocation"
 
 npm install
+npm version
```

Comment:
to understand what node version is being used when called from GH action without explicitly setting a version

Reply:
Probably intention was to use:
```suggestion
npm --version
```

npm version is available from npm 6. We are running this job in evergreen with npm version not managed by us so we tend to avoid using latest features.

Reply:
Context: only the minimal nodejs version set in the package.json matters. We do not care about the nodejs versions. Package versions are locked thru the package.lock file so adding this will be a noise and confuse others that npm/versions matter. In most of the cases, we going to print the same versions.

Reply:
For clarity I think what we intend is:
```suggestion
`echo "DEBUG: script is running using $(npm --version)"
```

Reply:
thank you, i was not aware that it was called from other projects, will leave only the asdf change

Reply:
All good. We can still add some improvements to scripts but we need to:

- Make sure that our change is aligned with the existing lowest npm version we support
- Have solid "why" for the change to minimize blast radius